### ISSUE TITLE
sklearn sgdregressor

### ISSUE THREAD

**Issue by chengsoonong** [chengsoonong/mclass-sky#133]
Implement a predictor for redshift using SGDRegressor
http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html
with huber loss

**Comment by jakubnabaglo** [comment]
See pull request #136. The R^2 value is (how??) -5.550428722967837e+25.

**Comment by chengsoonong** [comment]
Revisit after you have checked that R^2 is the same in your implementation and sklearn.

**Comment by jakubnabaglo** [comment]
Progress made in #154. R^2 is now 0.5639171254698758. This is more reasonable but still not as good as GP regression’s 0.9113252622189996.
